client/source/utilities/axiosRequests.js

```diff
@@ -1,18 +1,52 @@
 import axios from 'axios';
 
 
-const authServerURL = 'http://localhost:3000/api';
-
+const authServerURL = 'http://localhost:3000';
 
+/**
+ * Fire an Axios Get Request.
+ *
+ * It takes a string value to append to the endpoint,
+ * which should match the Atelier API endpoint pattern.
+ * @param {string} endpoint - Endpoint to be used
+ * @returns an Axios Request Promise
+ * */
 const get = function getByEndpoint(endpoint) {
-  return axios.get(`${authServerURL}/${endpoint}`);
+  return axios.get(`${authServerURL}/api/${endpoint}`);
 };
 
-const post = function postBygetByEndpoint(endpoint) {
-  return axios.post(`${authServerURL}/${endpoint}`);
+/**
+ * Fire an Axios Post Request.
+ *
+ * It takes a data json, and a string value to append to the endpoint,
+ * which should match the Atelier API endpoint pattern.
+ * @param {string} endpoint - Endpoint to be used
+ * @param {json} json - Data to be posted
+ * @returns an Axios Request Promise
+ * */
+const post = function postBygetByEndpoint(endpoint, json) {
```

Comment:
I saw one of your commit comments on not including a POST. Do you need a test example? I got one to work with posting a review. Just lmk!

Reply:
Yes please! It's been bothering me that I can't get the POST changes to be present in my later GET requests. I must have malformed data somehow.